pkg/controller/performanceprofile/performanceprofile_controller_test.go

```diff
@@ -249,6 +256,12 @@ var _ = Describe("Controller", func() {
 			result, err = r.Reconcile(request)
 			Expect(err).ToNot(HaveOccurred())
 			Expect(result).To(Equal(reconcile.Result{}))
+
+			// verify creation event
+			fakeRecorder, ok := r.recorder.(*record.FakeRecorder)
+			Expect(ok).To(BeTrue())
+			event := <-fakeRecorder.Events
+			Expect(event).To(ContainSubstring("Creation succeeded"))
```

Comment:
can we get a func test that ensures that if all objects already exist, and the reconcile loop is poped, that no events are triggered?

Reply:
func or unit test?

Reply:
done

Reply:
oh, i meant unit. sorry. you figured it out though :+1: